mev-identification-bot-py/src/findings.py

```diff
@@ -24,7 +27,12 @@ def MEVAccount(address: str, transfer_event_count: int, unique_token_count: int,
             'alert_id': 'MEV-ACCOUNT',
             'type': FindingType.Info,
             'severity': FindingSeverity.Info,
-            'metadata': {'transfer_event_count': transfer_event_count, 'unique_token_count': unique_token_count, 'unique_contract_address_count': unique_contract_address_count},
+            'metadata': {'anomaly_score': calculate_alert_rate(
+                        chain_id,
+                        BOT_ID,
+                        'MEV-ACCOUNT',
+                        ScanCountType.TRANSFER_COUNT,
```

Comment:
Could you use `ScanCountType.TX_WITH_INPUT_DATA_COUNT` for the denominator?
https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#LL21C5-L21C29